test/trust-proxy.test.js

```diff
@@ -5,23 +5,46 @@ const test = t.test
 const sget = require('simple-get').concat
 const fastify = require('..')
 
+const sgetForwardedRequest = (app, forHeader, path) => {
+  sget(
+    {
```

Comment:
can you please move { to the previous line and reformat the rest of the indentation?

Reply:
Done